ckan/setup/server_start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Doesn't need to be bullet proof. Should be easy to read.
```suggestion
DIR="$(dirname "${BASH_SOURCE[0]}")"
```